src/coap_prng.c

```diff
@@ -71,13 +71,13 @@ coap_set_prng(coap_rand_func_t rng) {
 }
 
 void
-coap_prng_init(unsigned long seed) {
+coap_prng_init(uint64_t seed) {
```

Comment:
Changing the type to `uint64_t` just to cast to `unsigned int` below seems weird. I suggest using `unsigned int` for `seed` to make transparent what is happening here.

Reply:
Made it an `unsigned int `flowing down from `coap_startup()`.